### PR TITLE
Fix coala/landing-frontend#189

### PR DESCRIPTION
**Pull request by raghavmittal101**
Added `overflow-x` property to `.coafile` .

### CODE REVIEW

**Comment by hemangsk** [comment]
Please get assigned to the issue you are fixing, for that you can go on gitter and say Hello World, you will get an email invite to join coala org :)

**Comment by raghavmittal101** [comment]
Thank you for kind help. I have followed the steps you suggested and assigned this issue to myself.

File: coala.css [hunk 1]
```diff
@@ -773,6 +773,10 @@ a.chip i {
   padding: 0;
 }
 
+.coafile {
+  overflow-x: auto;
+}
+
 .coala-online .coafile {
   background: #263238;
   white-space: pre-line;
```
Comment (hemangsk) [inline coala.css]: overflow-x: auto should go inside the ```.coala-online .coafile```  selector

**Review by hemangsk — CHANGES_REQUESTED** [review]

**Review by hemangsk — APPROVED** [review]

**Comment by hemangsk** [comment]
@raghavmittal101 awesome, just one more request to squash both of the commits into one, and to follow the commit message guidelines on https://coala.io/commit

**Comment by Arsfiqball** [comment]
@raghavmittal101 you need to squash your commit.

**Comment by raghavmittal101** [comment]
I have created a new pull request from another branch here https://github.com/coala/coalaCSS/pull/16. Please consider that one. Sorry for creating all this mess and wasting precious time of the community members.

**Comment by yukiisbored** [comment]
Closing for newer PR https://github.com/coala/coalaCSS/pull/16. Also, @raghavmittal101 here's a tip GitHub (and GitLab as well) doesn't care about the branch at all it just only see the name, You can just copy your work over to another branch, destroy the old one that's being used in the PR, create a new one, cherry pick the commits from the other branch (you can modify them if you wish!), and do a force push to the same branch name. et Voila! no need to create new pull requests!

Feel free to ask us questions about git and the tools that we use on our gitter channels :)

**Comment by raghavmittal101** [comment]
@yukiisbored and @Arsfiqball Thank you for helping m. :) . I am using github since a couple of years, but never worked with opensource organizations like coala. Its a nice experience. :sweat_smile:

**Pull request closed without merging**